readme.md

```diff
@@ -36,6 +37,64 @@ i18n.t('my.key'); // I'm a sentence
 i18n.t(['my.missing.key', 'my.key']); // I'm a sentence
 ```
 
+### Translate with templates
+There are several templates that are used by default:
```

Comment:
let's better use this sentence instead: `There are several predefined templates you can use`. WDYT?